src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/IStringBuilderConverter.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text;
+using JetBrains.Annotations;
+
+namespace RegressionGames.StateRecorder
+{
+    public interface IStringBuilderConverter
+    {
+
+        public void WriteToStringBuilderNullable(StringBuilder stringBuilder, [CanBeNull] object val)
```

Comment:
New interface that does what `IStringBuilderWriteable` used to do.  This is used for converters of json and is better named.